src/SettingsRepositories/DatabaseSettingsRepository.php

```diff
@@ -71,6 +73,25 @@ public function updatePropertyPayload(string $group, string $name, $value): void
             ]);
     }
 
+    public function updatePropertiesPayload(string $group, Collection $properties): void
+    {
+        $propertiesInBatch = (clone $properties)->map(function ($property) {
+            $property['payload'] = json_encode($property['payload']);
+
+            return $property;
+        })->toArray();
+
+        try {
+            $this->getBuilder()
```

Comment:
Let us let it fail hard, no need to catch this.

Reply:
Done